### PR TITLE
Add menu entry to directly collapse posts

### PR DESCRIPTION
**Pull request by annando**
I just discovered that we currently only could collapse posts of an author on their contact page and not directly from the post itself. This is now fixed.

### CODE REVIEW

File: view/theme/frio/templates/wall_thread.tpl [hunk 1]
```diff
@@ -404,7 +404,12 @@ as the value of $top_child_total (this is done at the end of this file)
                                                 <a class="btn-link navicon ignore" href="javascript:ignoreAuthor('item/ignore/{{$item.id}}/{{$item.return}}', 'item-{{$item.guid}}');" title="{{$item.ignore_author.ignore}}"><i class="fa fa-ban" aria-hidden="true"></i> {{$item.ignore_author.ignore}}</a>
                                         </li>
                                         {{/if}}
-                                    </ul>
+                                        {{if $item.collapse}}
+										<li role="menuitem">
+												<a class="btn-link navicon collapse" href="javascript:collapseAuthor('item/collapse/{{$item.id}}/{{$item.return}}', 'item-{{$item.guid}}');" title="{{$item.collapse.collapse}}"><i class="fa fa-ban" aria-hidden="true"></i> {{$item.collapse.collapse}}</a>
+										</li>
+										{{/if}}
+									</ul>
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Please normalize the indentation in this file. The rest of the file uses spaces.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
messages.po check failed.

**Comment by annando** [comment]
Yeah, but I don't know why.

**Comment by MrPetovan** [comment]
🤷‍♂️
```
+ git init -b 2023.05-rc
Initialized empty Git repository in /woodpecker/src/github.com/friendica/friendica/.git/
+ git remote add origin https://github.com/friendica/friendica.git
+ git fetch --no-tags --depth=1 --filter=tree:0 origin +refs/pull/13142/merge:
From https://github.com/friendica/friendica
 * branch            refs/pull/13142/merge -> FETCH_HEAD
+ git checkout -qf FETCH_HEAD
fatal: unable to access 'https://github.com/friendica/friendica.git/': Could not resolve host: github.com
fatal: could not fetch 7b6eecb66e14346f531d33334f8289a196aa6abc from promisor remote
exit status 128
```

**Pull request merged**